base/cpuid_test.cpp

```diff
@@ -16,7 +17,8 @@ TEST_F(CPUIDTest, Vendor) {
   // This mostly checks that we are getting something from CPUID, since it is
   // hard to expect things from the feature flags.  This could be expanded to an
   // AnyOf as needed if the tests are run on non-Intel processors.
-  EXPECT_THAT(CPUVendorIdentificationString(), Eq("GenuineIntel"));
+  EXPECT_THAT(CPUVendorIdentificationString(),
+              AnyOf(Eq("AuthenticAMD"), Eq("GenuineIntel")));
```

Comment:
I think it would read better without the Eq in this case: `EXPECT_THAT(CPUVendorIdentificationString(), AnyOf("AuthenticAMD", "GenuineIntel"));`